src/devices/Hmc5883l/Register.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Iot.Device.Hmc5883l
+{
+    /// <summary>
+    /// Register of HMC5883L
+    /// </summary>
+    public enum Register : byte
```

Comment:
internal - this is an implementation detail

Reply:
@krwq  If so, the code in master need to be changed

Reply:
@ZhangGaoxing not sure what do you mean

Reply:
I don't understand the comment either, but I agree that this should be internal unless you expose it, which doesn't seem to be the case.